docs/reference/remap/literals/float.cue

```diff
@@ -8,6 +8,8 @@ remap: literals: float: {
 
 		A decimal floating-point literal consists of an integer part (decimal digits), a decimal point, a fractional
 		part (decimal digits).
+
+		The maximum value for floags in VRL is 1.7976931348623157E+308f64. If a float exceeds that, the value is wrapped.
```

Comment:
```suggestion
		The maximum value for floats in VRL is 1.7976931348623157E+308f64. If a float exceeds that, the value is wrapped.
```

We may want to put the min in here too to describe a range.

Reply:
"floags" uggggh. Also good call on the range.